src/ophys_etl/modules/suite2p_registration/__main__.py

```diff
@@ -36,21 +36,47 @@ def run(self):
         if suite2p_args['force_refImg'] and len(suite2p_args['refImg']) == 0:
             # Use our own version of compute_reference to create the initial
             # reference image used by suite2p.
-            self.logger.info('Creating initial reference image from '
-                             f'{suite2p_args["nimg_init"]} frames...')
+            self.logger.info(f'Loading {suite2p_args["nimg_init"]} frames '
+                             'for reference image creation.')
             intial_frames = load_initial_frames(
                 file_path=suite2p_args['h5py'],
                 h5py_key=suite2p_args['h5py_key'],
                 n_frames=suite2p_args['nimg_init'],)
             # Create the initial reference image and store it in the
```

Comment:
I feel like this comment is now more appropriate to the `else` block where you call `compute_reference` directly. Please move it there.

Reply:
Sorry? There is a "Creating initial..." log in the `do_optimize_motion_params` else block. This one sends to the log regardless as it's at the start of the if `force_refImg` block.

Reply:
I feel like, if I were looking at the code for the first time, I would be confused by the "8 iterations is the Suite2P default"  statement, since the call to `optimize_motion_parameters` doesn't include any specification of `n_iterations`.

(Which raises an interesting point: should we be passing `max_reference_iterations` through to `optimize_motion_parameters`? I'm honestly not sure, I suspect "yes," if just to make sure that the code does what we say it does).

Reply:
Oh, sorry. The way github displayed this distracted me from what you were saying. Thought it was about the code change on the comment below it.